src/components/ActiveTeamsComponent.jsx

```diff
@@ -11,84 +9,69 @@ const ActiveTeamsComponent = ({
   handleDragStart,
   handleDragOver,
   handleDragDrop,
-}) => {
-  const myListRef = useRef(null);
-  const [isEmpty, setIsEmpty] = useState(true);
-
-  useEffect(() => {
-    if (myListRef.current === null) {
-      setIsEmpty(true);
-    } else if (myListRef.current.getElementsByTagName('li').length < 1) {
-      setIsEmpty(true);
-    } else {
-      setIsEmpty(false);
-    }
-  }, [teams]);
```

Comment:
This part is not needed because the only thing you're checking is if the list of teams is filled or not. If you pass only relevant teams into the component, this state handling is obsolete.

Reply:
Truly Noted.